src/main/java/seedu/savenus/ui/InfoWindow.java

```diff
@@ -14,15 +14,18 @@
 import seedu.savenus.model.info.BudgetInfo;
 import seedu.savenus.model.info.BuyInfo;
 import seedu.savenus.model.info.ClearInfo;
```

Comment:
I think the clear command destroys everything not only the Menu right now, except for the Wallet (Which will also be cleared along with the rest in the my next PR).